webrisk/cmd/wrserver/statik/statik.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 Google LLC
+// //
```

Comment:
Extra slashes are messing up the license check. Same with webrisk.pb.go.

Reply:
Oh, my bad, sorry